pkg/mimirtool/commands/alerts.go

```diff
@@ -125,15 +126,28 @@ func (a *AlertmanagerCommand) readAlertManagerConfig() (string, map[string]strin
 		return "", nil, err
 	}
 
+	templates, err := a.readAlertManagerConfigTemplates()
+	if err != nil {
+		return "", nil, err
+	}
+
+	return cfg, templates, nil
+}
+
+func (a *AlertmanagerCommand) readAlertManagerConfigTemplates() (map[string]string, error) {
 	templates := map[string]string{}
 	for _, f := range a.TemplateFiles {
 		tmpl, err := os.ReadFile(f)
 		if err != nil {
-			return "", nil, errors.Wrap(err, "unable to load template file: "+f)
+			return nil, errors.Wrap(err, "unable to load template file: "+f)
 		}
-		templates[f] = string(tmpl)
+		name := filepath.Base(f)
+		if _, ok := templates[name]; ok {
+			return nil, errors.New("cannot have multiple templates with same file names but different paths: "+f)
```

Comment:
it's a shame we can't tell the user which files exactly are conflicting. I suppose just the base name should be enough

Reply:
Good idea. I'll just add a little code to also track the original names.